src/main/java/com/powsybl/sld/server/SingleLineDiagramController.java

```diff
@@ -273,4 +274,43 @@ public ResponseEntity<Collection<String>> getAvailableSvgComponentLibraries() {
         }
         return networkAreaDiagramService.getNetworkAreaDiagramSvgAsync(networkUuid, variantId, voltageLevelsIds, depth, withGeoData);
     }
+
+    @PostMapping(value = "/network-area-diagram/config/", consumes = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(summary = "Create a network area diagram config")
+    @ApiResponse(responseCode = "200", description = "The network area diagram config has been created")
+    public ResponseEntity<UUID> createNetworkAreaDiagramConfig(@RequestBody NadConfigInfos nadConfigInfos) {
+        return ResponseEntity.ok().body(networkAreaDiagramService.createNetworkAreaDiagramConfig(nadConfigInfos));
+    }
+
+    @PutMapping(value = "/network-area-diagram/config/{nadConfigUuid}", consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(summary = "Update a network area diagram config")
+    @ApiResponses(value = {
+        @ApiResponse(responseCode = "200", description = "The network area diagram config was updated"),
+        @ApiResponse(responseCode = "404", description = "The network area diagram config was not found"),
+    })
+    public ResponseEntity<Void> updateNetworkAreaDiagramConfig(
+            @Parameter(description = "Network Area Diagram UUID") @PathVariable("nadConfigUuid") UUID nadConfigUuid,
```

Comment:
```suggestion
            @Parameter(description = "Network Area Diagram configuration UUID") @PathVariable("nadConfigUuid") UUID nadConfigUuid,
```

same for the 3 endpoints below

Reply:
Good catch